tests/clang_src2src/pragma_sst_loop_count.cc

```diff
@@ -11,10 +11,11 @@ int fxn()
   double* x = new double[10];
   int* idx = new int[5];
   int loopMax = 5;
+  int sstLoopCount = loopMax/2;
 #pragma sst compute
   for (int i=0; i < loopMax; ++i){
     mul *= i;
-    #pragma sst loop_count loopMax/2
+    #pragma sst loop_count sstLoopCount
```

Comment:
Why the change here?

Reply:
Loop count pragmas are much more limited in what they can be. They can only be a constant or variable name, not a whole expression.

Reply:
Although this limits somewhat flexibility in the skeleton, this allows us to directly modify the AST and greatly simplifies logic in the rewriter.